Stripe/STPPaymentCardTextField.m

```diff
@@ -74,10 +77,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
 }
 
 - (void)commonInit {
-    
-    self.borderColor = [self.class placeholderGrayColor];
-    self.cornerRadius = 5.0f;
-    self.borderWidth = 1.0f;
+    _borderColor = [self.class placeholderGrayColor];
+    _cornerRadius = 5.0f;
+    _borderWidth = 1.0f;
```

Comment:
Since we're just passing through to the layer, can you just not have an instance variable and pass the setters and getters through, or does that not work with UIAppearance?